src/context/AppProvider/AppContext.tsx

```diff
@@ -365,6 +375,48 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     dispatch(marketApi.endpoints.findByAssetId.initiate(routeAssetId))
   }, [dispatch, routeAssetId])
 
+  useEffect(() => {
+    if (!isPortfolioLoaded || !wallet || !supportsETH(wallet)) return
+    // getting fox-eth lp token data
+    const {
+      getFoxEthLpGeneralData,
+      getFoxEthLpWalletData,
+      getFoxFarmingContractGeneralData,
+      getFoxFarmingContractWalletData,
+    } = foxEthApi.endpoints
+    dispatch(getFoxEthLpGeneralData.initiate())
```

Comment:
Can we use `useGetFox(Eth)?XYZQuery` hooks instead? This is within the context of `<ReduxProvider />` so we should be able to.